src/components/CategoriesBrowser/CategoriesBrowserPresentation.js

```diff
@@ -55,7 +55,7 @@ function CategoriesBrowserPresentation({
                 <h2 className="w-1/2 ml-4">Category</h2>
                 <h2 className="w-1/2">Parent Category</h2>
               </div>
-              <ul className="flex-col divide-y divide-gray-200 bg-white overflow-y-scroll grow mt-2 h-72">
+              <ul className="flex-col divide-y divide-gray-200 bg-white overflow-y-scroll grow mt-2 pt-2 h-72">
```

Comment:
I also noticed an issue where the border of the top element was hidden, this fixes that.